content/100-getting-started/03-prisma-postgres/115-import-from-mysql-database.mdx

```diff
@@ -0,0 +1,346 @@
+
+### If you **were not** previously using Prisma ORM
+
+Initialize Prisma in your project by running `npx prisma init` in your project directory. This creates a `prisma` folder with a `schema.prisma` file and `.env` file (if not already present).
+
+In the generated `.env` file, update `DATABASE_URL` to match your Prisma Postgres connection string that you received in [step 1](/getting-started/prisma-postgres/import-from-mysql-database#1-create-a-new-prisma-postgres-database):
+
+```terminal  file=.env no-copy
+DATABASE_URL="prisma+postgres://accelerate.prisma-data.net/?api_key=__API_KEY__"
+```
+
+[Introspect](/orm/prisma-schema/introspection) your newly migrated database by running:
+
+```terminal
+npx prisma db pull
+```
+
+This command updates your `schema.prisma` file with models representing your migrated tables, so you can start using [Prisma Client](/orm/prisma-client) to query your data or [Prisma Migrate](/orm/prisma-migrate/getting-started) to manage future changes.
+
+
+Congratulations! You've successfully migrated your MySQL database to Prisma Postgres and configured your Prisma project. Your migration tutorial is now complete.
+
+:::note
+
+For a comprehensive guide on getting started with Prisma and Prisma Postgres, see [start from scratch with Prisma and Prisma Postgres](/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-prismaPostgres).
+
+:::
+
+### If you **were** already using Prisma ORM
+
+In your `schema.prisma` file, change the `provider` in the `datasource` block from `mysql` to `postgresql`:
+
+```prisma file=schema.prisma
+datasource db {
+  // delete-start
+  provider = "mysql"
+  // delete-end
+  // add-start
+  provider = "postgres"
+  // add-end
+  url      = env("DATABASE_URL")
+}
+```
+
+In the generated `.env` file, update `DATABASE_URL` to match your new Prisma Postgres connection string that you received in [step 1](/getting-started/prisma-postgres/import-from-mysql-database#1-create-a-new-prisma-postgres-database):
+
+```terminal file=.env no-copy
+DATABASE_URL="prisma+postgres://accelerate.prisma-data.net/?api_key=__API_KEY__"
+```
+
+Introspect your newly migrated Prisma Postgres database and generate Prisma Client:
+
+```terminal
+npx prisma db pull
+```
+
+This command refreshes your Prisma models based on the new database schema.
+
+If you were using [Prisma Migrate](/orm/prisma-migrate/getting-started) before:
+
+- Delete your existing `migrations` folder in the `prisma` directory.
+- [Baseline your database](/orm/prisma-migrate/workflows/baselining#baselining-a-database) to begin creating new migrations.
+
+Congratulations! You've successfully migrated your MySQL database to Prisma Postgres and configured your Prisma project. Your migration tutorial is now complete.
+
+## Migrating data from PlanetScale to Prisma Postgres
```

Comment:
do we need an entirely different section for this? I actually didn't see this section before and just followed the other instructions which worked fine except for the SSL config issue. if you think that PlanetScale is substantially different, I'd suggest putting it into another guide using the DB switcher so that we'd have three guides in total that can be selected under the DB switcher under **Import from existing database**:
- PostgreSQL
- MySQL
- PlanetScale